serverless/containers/concepts.mdx

```diff
@@ -138,4 +138,4 @@ Unit used to measure the resource consumption of a container. It reflects the am
 
 ## Protocol
 
-Serverless Containers supports **http1** (default) and **http2** (`h2c`). In some cases, for example, while deploying a container using gRPC, you may need to upgrade the protocol for it to work.
+Serverless Containers supports **http1** (default) and **http2** (`h2c`). Use HTTP/2 if your container application is configured to listen for HTTP/2 requests, such as a gRPC service or a web server that utilizes HTTP/2 features like multiplexing, otherwise HTTP/1 is recommended.
```

Comment:
```suggestion
Serverless Containers supports **http1** (default) and **http2** (`h2c`). Use HTTP/2 if your container application is configured to listen for HTTP/2 requests, such as a gRPC service or a web server that uses HTTP/2 features like multiplexing, otherwise HTTP/1 is recommended.
```